README.md

```diff
@@ -1115,21 +1116,68 @@ Reserved for future use. Currently both the 'postgres' and 'redshift' dialects a
 
 Default value: inherit from server settings.
 
+##### `ensure`
+
+Specifies whether to ensure the group exists or is removed.
+
+Valid options: 'present' or 'absent'.
+
+Default value: 'present'.
+
 ##### `groupname`
 Defines the name of the group to create.
 
 Default value: the namevar.
 
-##### `groupmembers`
-Defines the users that are part of the current group, if any.
+##### `port`
+Optional port override for connecting to postgres when applying this group.
+
+Default value: inherit from `$connect_settings` or `postgresql::server::port`
+
+#### postgresql::server::dbgroupmember
+
+Creates a member for an existing Postgres group.
+
+##### `connect_settings`
+Required.
+
+Specifies a hash of environment variables used when connecting to a remote server.
+
+Default value: `undef`, because groups only currently make sense in remotely-managed Redshift clusters.
```

Comment:
I'm not sure I understand this default justification. If these are ENV variables that the psql cli uses (https://www.postgresql.org/docs/8.4/static/libpq-envars.html), can you mention that?

Reply:
Fixed.